From de2a8e503d534602103935553bbe45aa02081357 Mon Sep 17 00:00:00 2001
From: Roman Li <Roman.Li@amd.com>
Date: Thu, 17 Aug 2017 16:14:45 -0400
Subject: [PATCH 3173/5855] drm/amd/display: Fix FBC disable for stereo

Removed FBC disable from dc_enable_stereo().
It should be implemented in stereo_setup callback, which is not
implemented yet. Otherwise it always disable FBC regardless
if stereo supported or not.

Signed-off-by: Roman Li <Roman.Li@amd.com>
Reviewed-by: Bhawanpreet Lakha <Bhawanpreet.Lakha@amd.com>
Acked-by: Harry Wentland <Harry.Wentland@amd.com>
---
 drivers/gpu/drm/amd/display/dc/core/dc.c | 11 -----------
 1 file changed, 11 deletions(-)

diff --git a/drivers/gpu/drm/amd/display/dc/core/dc.c b/drivers/gpu/drm/amd/display/dc/core/dc.c
index 5714c9b..2976300 100644
--- a/drivers/gpu/drm/amd/display/dc/core/dc.c
+++ b/drivers/gpu/drm/amd/display/dc/core/dc.c
@@ -883,10 +883,6 @@ bool dc_enable_stereo(
 	struct pipe_ctx *pipe;
 	struct dc  *core_dc = dc;
 
-#ifdef ENABLE_FBC
-	struct compressor *fbc_compressor = core_dc->fbc_compressor;
-#endif
-
 	for (i = 0; i < MAX_PIPES; i++) {
 		if (context != NULL)
 			pipe = &context->res_ctx.pipe_ctx[i];
@@ -899,13 +895,6 @@ bool dc_enable_stereo(
 		}
 	}
 
-#ifdef ENABLE_FBC
-	if (fbc_compressor != NULL &&
-	    fbc_compressor->funcs->is_fbc_enabled_in_hw(core_dc->fbc_compressor,
-							NULL))
-		fbc_compressor->funcs->disable_fbc(fbc_compressor);
-
-#endif
 	return ret;
 }
 
-- 
2.7.4

